dockerfiles/Dockerfile.server

```diff
@@ -0,0 +1,18 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+# Official user quickstart docker container for ONNX Runtime
+# Ubuntu 16.04, CPU version, Python 3.
```

Comment:
where is python used?

Reply:
Used python in the new update version build process